lib/components/radio/radio.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+import 'package:mix/mix.dart';
+
+import 'radio.style.dart';
+import 'radio.variants.dart';
+
+class RemixRadio extends StatelessWidget {
+  const RemixRadio({
+    super.key,
+    this.label,
+    this.disabled = false,
+    this.active = false,
+    this.onChanged,
+    RadioStyles? style,
+  }) : _customStyle = style;
+
+  final String? label;
+  final bool disabled;
+  final bool active;
+  final ValueChanged<bool>? onChanged;
+
+  final RadioStyles? _customStyle;
+
+  @override
+  Widget build(BuildContext context) {
+    var variant = active ? RadioState.active : RadioState.inactive;
+
+    final style =
+        RadioStyles.defaults().merge(_customStyle).selectVariants([variant]);
+
+    return PressableBox(
+      onPressed:
+          onChanged == null || disabled ? null : () => onChanged!(!active),
+      child: HBox(
+        style: style.row,
```

Comment:
Are we going to keep the consistency of function syntax? Example `style.row()` instead?

Reply:
I've developed a new solution for Styles classes. If all styles must be functions, the work of Styles classes will become simpler. We won't need to extend StyleRecipe anymore, and overriding styles will be easier by extending the Styles classes.

```dart
class RadioStyles {
  const RadioStyles();

  Style outerContainer(List<Variant> variants) => Style(
        box.width(16),
        box.height(16),
        box.alignment.center(),
        box.borderRadius.all(10),
        box.color.transparent(),
        box.border.all(width: 1.5, color: Colors.black87),
      );

  Style innerContainer(List<Variant> variants) => Style(
        box.borderRadius.all(10),
        box.color.black87(),
        RadioState.active(
          box.width(8.5),
          box.height(8.5),
        ),
        RadioState.inactive(
          box.width(0),
          box.height(0),
        ),
      );

  Style label(List<Variant> variants) => Style(
        text.style.fontSize(16),
        text.style.bold(),
        text.style.color.black87(),
      ).applyVariants(variants);

  Style row(List<Variant> variants) => Style(
        flex.mainAxisAlignment.center(),
        flex.crossAxisAlignment.center(),
        flex.mainAxisSize.min(),
        flex.gap(6),
      );
}

class RadioStylesDefaults extends RadioStyles {
  const RadioStylesDefaults();

  @override
  Style outerContainer(List<Variant> variants) {
    return super.outerContainer(variants);
  }

  @override
  Style innerContainer(List<Variant> variants) {
    return super.innerContainer(variants);
  }

  @override
  Style label(List<Variant> variants) {
    return super.label(variants);
  }

  @override
  Style row(List<Variant> variants) {
    return super.row(variants);
  }
}
```